livekit_analytics.proto

```diff
@@ -152,3 +153,27 @@ message AnalyticsEvent {
 message AnalyticsEvents {
   repeated AnalyticsEvent events = 1;
 }
+
+message AnalyticsRoomParticipant {
+  string id = 1;
+  string identity = 2;
+  string name = 3;
+  ParticipantInfo.State state = 4;
+  ParticipantInfo.Kind kind = 5;
+  google.protobuf.Timestamp joined_at = 6;
+}
+
+message AnalyticsRoom {
+  string id = 1;
+  string name = 2;
+  google.protobuf.Timestamp created_at = 3;
+  uint32 num_participants = 4;
```

Comment:
what do we use these for?

Reply:
will remove, its redundant, we can infer from message